app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -156,15 +164,15 @@ ja-JP=Japanese (Japan)
 ko-KR=Korean
 lastContribution=Last contribution:
 lastPass=LastPass&reg;
-ledgerBackupContent=Below, you will find the anonymized recovery keys that are required if you ever lose access to this computer.
+ledgerBackupContent=Below, you will find the anonymized recovery key that is required if you ever lose access to this computer.
```

Comment:
This needs some text to indicate that the recovery key should not be shared or posted publicly (like what we do for the sync passphrase)

Reply:
@diracdeltas any suggestions on the copy?

Reply:
bold font: "Make sure you keep this key private, or else your wallet will be compromised." ?

Reply:
@diracdeltas we have text in backup file as well `Note: This key is not stored on Brave servers. This key is your only method of recovering your Brave wallet. Save this key in a safe place, separate from your Brave browser.` should we modify it?

Reply:
will add the same text that we added in the dialog

Reply:
done

Reply:
lgtm!